### PR TITLE
Include 2024 Draft Note publication

### PR DESCRIPTION
**Pull request by daniel-montalvo**

### CODE REVIEW

**Review by maryjom — APPROVED** [review]

**Review by remibetin — COMMENTED** [review]
@daniel-montalvo @shawna-slh I recommend using a list, as in the other Task force pages. 

Since the two documents are closely related, I would also suggest clarifying the differences between the two. For example, will the 2024 version supersede the 2013 version when published as a W3C Working Group Note? I leave this to your appreciation.

**Comment by daniel-montalvo** [comment]
> @daniel-montalvo @shawna-slh I recommend using a list, as in the other Task force pages.

Will do.

> Since the two documents are closely related, I would also suggest clarifying the differences between the two. For example, will the 2024 version supersede the 2013 version when published as a W3C Working Group Note? I leave this to your appreciation.

Also, will do. We may want to discuss references to year, based on @shawna-slh  comments in another thread.

File: pages/about/groups/task-forces/wcag2ict.md [hunk 1]
```diff
@@ -70,7 +70,10 @@ The WCAG2ICT TF uses the public-wcag2ict-tf@w3.org mailing list ([mailing list a
 
 ## Publications
 
-2013 W3C Working Group Note [Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict/)
+- W3C Working Group Note [Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict/) &mdash; Provides guidance on WCAG 2.0 success criteria
+- W3C Draft Working Group Note [Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/) &mdash; Provides guidance on 2.0, 2.1, and 2.2 success criteria, will superceed the previous Note.
+
+2024 W3C Draft Working Group Note [Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/)
```
Comment (remibetin) [inline pages/about/groups/task-forces/wcag2ict.md:76]: @daniel-montalvo Isn't this sentence redundant?

File: pages/about/groups/task-forces/wcag2ict.md [hunk 2]
```diff
@@ -70,7 +70,10 @@ The WCAG2ICT TF uses the public-wcag2ict-tf@w3.org mailing list ([mailing list a
 
 ## Publications
 
-2013 W3C Working Group Note [Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict/)
+- W3C Working Group Note [Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict/) &mdash; Provides guidance on WCAG 2.0 success criteria
+- W3C Draft Working Group Note [Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/) &mdash; Provides guidance on 2.0, 2.1, and 2.2 success criteria, will superceed the previous Note.
```
Comment (remibetin) [inline pages/about/groups/task-forces/wcag2ict.md:74]:
```suggestion
- W3C Draft Working Group Note [Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/) &mdash; Provides guidance on 2.0, 2.1, and 2.2 success criteria, will supersede the previous Note.
```

File: pages/about/groups/task-forces/wcag2ict.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ The WCAG2ICT TF uses the public-wcag2ict-tf@w3.org mailing list ([mailing list a
 ## Publications
 
 - W3C Working Group Note [Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict/) &mdash; Provides guidance on WCAG 2.0 success criteria
-- W3C Draft Working Group Note [Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/) &mdash; Provides guidance on 2.0, 2.1, and 2.2 success criteria, will superceed the previous Note.
+- W3C Draft Working Group Note [Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/) &mdash; Provides guidance on 2.0, 2.1, and 2.2 success criteria, will supersede the previous Note.
 
 2024 W3C Draft Working Group Note [Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/)
```
Comment (maryjom) [inline pages/about/groups/task-forces/wcag2ict.md:76]:
```suggestion
```

**Comment by shawna-slh** [comment]
@remibetin ready to publish.

**Pull request merged**